examples/cartesian_path.py

```diff
@@ -76,16 +81,36 @@
 visualize_frames(viz, "cartesian_plan", tforms_to_show, line_length=0.05, line_width=1)
 time.sleep(0.5)
 
+curr_time = 0
+
 plt.ion()
 plt.figure()
-plt.title("Joint Position Trajectories")
+plt.title("Pyroboplan - Joint Position Trajectories")
 for idx in range(q_vec.shape[0]):
     plt.plot(t_vec, q_vec[idx, :])
-
-curr_time = 0
 time_line = plt.axvline(x=curr_time, color="k", linestyle="--")
+plt.legend(model.names[1:])
 
+# TOPPRA
+vlims = np.ones(q_start.shape[0]) * max_linear_velocity
+alims = np.ones(q_start.shape[0]) * max_linear_acceleration
+# Define the geometric path and two constraints.
+path = ta.SplineInterpolator(t_vec, q_vec.T)
+pc_vel = constraint.JointVelocityConstraint(vlims)
+pc_acc = constraint.JointAccelerationConstraint(alims)
```

Comment:
I don't think you want to be using Cartesian velocity limits here, right?

You probably want to be using the *joint* velocity limits from the Pinocchio model. And acceleration is something we have to make up since it's not part of the URDF.

For example, I changed the code to say

```python
vlims = model.velocityLimit
alims = 10* np.ones_like(vlims)
```

and now the trajectories are actually *faster*, not slower, with TOPP-RA.

![image](https://github.com/user-attachments/assets/9b9706ac-f5ad-4937-989a-49bec8469e61)

Reply:
Yeah, thanks. I will check that